orcid-web/src/main/resources/freemarker/includes/ng2_templates/edit-table-ng2-template.ftl

```diff
@@ -118,6 +118,32 @@
                     </td>
                 </tr>
             </@orcid.checkFeatureStatus>
+            <@orcid.checkFeatureStatus 'GET_MY_DATA'>
+                <tr>
+                    <th><a name="getMyData"></a>${springMacroRequestContext.getMessage("manage.get_my_data")}</th>
+                    <td><a href="" (click)="toggleGetMyDataEdit()">{{twoFAToggleText}}</a></td>
+                </tr>
+                <tr *ngIf="showEditGetMyData" >
+                    <td colspan="2">
+                        <p>
+                            <@orcid.msg 'manage.get_my_data.details.p1' />
+                        </p>
+                        <p>
+                            <@orcid.msg 'manage.get_my_data.details.p2' /> <a href="https://support.orcid.org/knowledgebase/articles/117225"><@orcid.msg 'common.learn_more' /></a>
```

Comment:
@amontenegro please use the ${knowledgeBaseUri} var in place of https://support.orcid.org/knowledgebase